Sources/StreamChat/Controllers/ChannelController/ChannelController_Tests.swift

```diff
@@ -2070,6 +2182,63 @@ class ChannelController_Tests: StressTestCase {
         AssertAsync.canBeReleased(&weakController)
     }
     
+    func test_sendKeystrokeEvent_noop() throws {
```

Comment:
I'm not sure an average iOS dev understands `noop` :) Can we change it to something more explicit like `test_sendKeystrokeEvent_whenTypingEventsAreDisabled_doesntEmitError`?